cnv/cnv_release_notes/cnv-release-notes.adoc

```diff
@@ -16,62 +16,122 @@ include::modules/technology-preview.adoc[leveloffset=+2]
+that had the same name as an existing virtual machine, the operation failed.
+This resulted in the message `Name is already used by another virtual machine`.
+This issue is fixed in {CNVProductName} 2.1.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1717802[*BZ#1717802*])
 
-* {CNVProductNameStart} cannot reliably identify node drains that are triggered by
-running either `oc adm drain` or `kubectl drain`. Do not run these commands on
-the nodes of any clusters where {CNVProductName} is deployed. The nodes might not
-drain if there are virtual machines running on top of them.
-The current solution is to put nodes into maintenance.
-(link:https://bugzilla.redhat.com/show_bug.cgi?id=1707427[*BZ#1707427*])
-
-* If you create a virtual machine with the Pod network connected in `bridge`
-mode and use a `cloud-init` disk, the virtual machine will lose its network
-connectivity after being restarted. As a workaround, remove the `HWADDR` line
-in the file `/etc/sysconfig/network-scripts/ifcfg-eth0`.
+* Previously, if you created a virtual machine with the Pod network connected in
+`bridge` mode and used a `cloud-init` disk, the virtual machine lost its network
+connectivity after being restarted. This issue is fixed in {CNVProductName} 2.1.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1708680[*BZ#1708680*])
 
-* Masquerade does not currently work with CNV. Due to an upstream issue,
-you cannot connect a virtual machine to the default Pod network while in Masquerade mode.
-(link:https://bugzilla.redhat.com/show_bug.cgi?id=1725848[*BZ#1725848*])
 
-* Creating a NIC with Masquerade in the wizard does not allow you to specify the `port` option.
-(link:https://bugzilla.redhat.com/show_bug.cgi?id=1725848[*BZ#1725848*])
+== Known issues
+
+* When adding a disk to a virtual machine via the *Disks* tab in the web console,
+the added disk always has a `Filesystem` volumeMode, regardless of the volumeMode
+set in the `kubevirt-storage-class-default` ConfigMap.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1753688[*BZ#1753688*])
+
+* After migration, a virtual machine is assigned a new IP address. However, the
+commands `oc get vmi` and `oc describe vmi` still generate output containing the
+obsolete IP address. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1686208[*BZ#1686208*])
++
+** As a workaround, view the correct IP address by running the following command:
++
+----
+$ oc get pod -o wide
+----
+
+* The virtual machines wizard does not load for users without administrator
+privileges. This issue is caused by missing permissions that allow users to load
+network attachment definitions.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1743985[*BZ#1743985*])
++
+** As a workaround, provide the user with permissions to load the network attachment
+definitions.
++
+. Define `ClusterRole` and `ClusterRoleBinding` objects to the YAML configuration
+file, using the following example:
++
```

Comment:
I haven't seen a `+` splitting object examples like this in our docs, but maybe we have. Should this be split into two code blocks?

Reply:
Good catch, I do think that was supposed to be two code blocks.